### DISCUSSION TITLE
Update Form - Number Fields Don't Display 0 as Value

### DISCUSSION THREAD

**Discussion by mmalessa80** [Budibase/budibase#10407] [Help] [answered]
Not sure if this is an internal setting I am missing somewhere, but I have two 'Number' type input fields in an Update form, neither will display 0 even though the fields have the correct binding. Any other '+' number will display the field value correctly when changed and then the row value after being saved. I added a screenshot of my row save payload next to the form after row save has completed. Any help would be appreciated, I certainly do need 0 to display in the respective fields. And a comparative screenshot when both fields contain a non-zero value.

![image](https://user-images.githubusercontent.com/106758851/234123727-71e324bf-68cb-4863-b582-65551c7f875f.png)

![image](https://user-images.githubusercontent.com/106758851/234124397-938a9c5f-20fd-4a55-8ad2-bd9773c073ff.png)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @mmalessa80,

I've replicated the issue you are experiencing and that can be found here: https://github.com/Budibase/budibase/issues/10412, there is something similar occurring with filtering were its not processing 0 correctly. That is in the works to be fixed within this iteration, my hope is that fixing that issue may also indirectly fix this one as well. If not the issue has been created and will be fixed!

I hope this helps!

**Reply by mmalessa80** [reply]
Thank you! Glad you were able to reproduce the issue. I look forward to seeing a fix hopefully.
